src/Compilers/VisualBasic/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.vb

```diff
@@ -137,6 +137,82 @@ True
         End Sub
 
         <Fact>
+        Public Sub MyTemplateNotCovered()
+            Dim testSource As XElement = <file name="c.vb">
+                                             <![CDATA[
+Module Program
+    Public Sub Main(args As String())                                   ' Method 1
+        TestMain()
+        Microsoft.CodeAnalysis.Runtime.Instrumentation.FlushPayload()
+    End Sub
+
+    Sub TestMain()                                                      ' Method 2
+    End Sub
+End Module
+]]>
+                                         </file>
+
+            Dim source As XElement = <compilation></compilation>
+            source.Add(testSource)
+            source.Add(InstrumentationHelperSource)
+
+            Dim expectedOutput As XCData = <![CDATA[
+Flushing
+8
+True
+True
+True
+9
+True
+12
+True
+True
+False
+True
+True
+True
+True
+True
+True
+True
+True
+]]>
+
+            Dim preprocessorSymbols = ImmutableArray.Create(New KeyValuePair(Of String, Object)("_MyType", "Console"))
+            Dim parseOptions = VisualBasicParseOptions.Default.WithPreprocessorSymbols(preprocessorSymbols)
```

Comment:
A comment explaining the effect of adding the preprocessor symbols might be appropriate, so that a casual scan of the test would reveal what it is testing.